index.html

```diff
@@ -12,283 +12,404 @@
+                    <img src="Pictures/snapshot.png" alt="">
                 </div>
-                <div class="headline-bio">
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
-                        of my
-                        work
-                        and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                        contact
-                        me.
-                    </p>
+                <h2>Tonic</h2>
+                <div class="bullets">
+                    <h3>CANOPY</h3>
+                    <ul class="bullets1">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
                 </div>
-                <div class="lets-connect">
-                    <h5>LET'S CONNECT</h5>
+                <div class="second-para">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 </div>
-                <div class="icons">
+                <div class="alltext">
                     <ul>
-                        <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
-                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
-                                    src="Pictures/Linkedin.png" alt=""></a></li>
-                        <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
-                        <li><a href="https://github.com/Boku52500"><img src="Pictures/github.png" alt=""></a></li>
-                        <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
+                        <li class="text1">html</li>
+                        <li class="text2">css</li>
+                        <li class="text3">javaScript</li>
                     </ul>
                 </div>
-            </div>
-    </section>
-
-
-    <main>
-
-        <section class="2-page">
-            <div class="snapshot">
-                <img src="Pictures/snapshot.png" alt="">
-            </div>
-            <h2>Tonic</h2>
-            <div class="bullets">
-                <h3>CANOPY</h3>
-                <ul class="bullets1">
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                </ul>
-            </div>
-            <div class="second-para">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            </div>
-            <div class="alltext">
-                <ul>
-                    <li class="text1">html</li>
-                    <li class="text2">css</li>
-                    <li class="text3">javaScript</li>
-                </ul>
-            </div>
-            <button class="but">See Project</button>
-        </section>
```

Comment:
- [ ] As the previous reviewer mention please follow figma template to match your design. kindly add an overall background color to your portfolio and give separate background color for every card and also give it a `border-radius`
## Your design👇
![card](https://user-images.githubusercontent.com/66992901/193365725-fee28278-f99d-48d7-9120-5042f2f65e3f.PNG)
 ## Figma template👇
![cardfigma](https://user-images.githubusercontent.com/66992901/193365750-5f07e74a-05f0-4006-be86-8e08f8c85571.PNG)